iroha/src/config.rs

```diff
@@ -106,11 +115,32 @@ mod tests {
     use std::collections::BTreeSet;
 
     const CONFIGURATION_PATH: &str = "tests/test_config.json";
+    const TRUSTED_PEERS_PATH: &str = "tests/test_trusted_peers.json";
 
     #[test]
     fn parse_example_json() -> Result<()> {
         let configuration = Configuration::from_path(CONFIGURATION_PATH)
             .wrap_err("Failed to read configuration from example config")?;
+            assert_eq!(
+                "127.0.0.1:1337",
+                configuration.torii_configuration.torii_p2p_url
+            );
+            assert_eq!(1000, configuration.sumeragi_configuration.block_time_ms);
+            Ok(())
+        }
+
+    #[test]
+    fn parse_example_trusted_peers_json() -> Result<(), String> {
```

Comment:
You can now use just bare `Result<T>`! It will convert all errors to the same type and you won't need to use `.map_err(|e| e.to_string())`. You can also add context by `.wrap_err("My context for error")` or `.wrap_err_with(|| format!("Here is some id {}", id))`.

Reply:
Thanks! ok, I update it 🙇